src/functions/blockchainFunctions.js

```diff
@@ -146,3 +146,21 @@ function exchangeFundsFlow_ (slug, from, to) {
     ]
   }))
 }
+
+function tokenTopTransactions_ (slug, from, to, limit) {
+  var results = getApiClient_().fetchTokenTopTransactions(slug, from, to, limit)
+  assertHasData_(results)
+  var headers = ['Date', 'From Address', 'From Is Exchange', 'To Address', 'To Is Exchange', 'TRX Hash', 'TRX Value']
```

Comment:
TRX Hash -> Transaction Hash
TRX Value -> Transacrtion Value

Reply:
It should be okay now